src/main/resources/checkstyle/hotels-checkstyle.xml

```diff
@@ -130,7 +132,7 @@
     <module name="ArrayTypeStyle" />
     <module name="Indentation">
       <property name="basicOffset" value="2" />
-      <property name="caseIndent" value="2" />
+      <property name="caseIndent" value="0" />
```

Comment:
Can you give an example of what this change looks like? Perhaps in one of the other PRs? Or does our Eclipse setup already do this and this just gets Checkstyle to match the Eclipse format?

Reply:
So this is already done by Eclipse with one of the imported files.

To give an example, something written with indent = 2 would look like
```
switch (method.getName()) {
  case "shutdown":
    return false;
  default:
    return true;
}
```

but even if that exists in our codebase, eclipse will refactor it to indent=0:
```
switch (method.getName()) {
case "shutdown":
  return false;
default:
  return true;
}
```

Reply:
So yeah, this change is to get Checkstyle to match our standards.

Reply:
Yeah, Eclipse can be changed to do that. 

But also, I found a discussion that argues why switch cases aren't indented: https://stackoverflow.com/questions/4509039/why-the-strange-indentation-on-switch-statements. I thought it might be useful to see why the Oracle code conventions opt for no indentation.

Reply:
My preference is for no indent on the case line. I think that having indentation for the switch block and then each case entry adds too much horizontal spacing. Roll on JEP-325 with better switch syntax.

Reply:
Yeah, true about all the additional spacing and also see the point about them being labels. OK, I'm fine with this change.